packages/components/src/truncate/test/truncate.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import React from 'react';
```

Comment:
React imports aren't necessary in Gutenberg since it's all handled by Babel. We should remove them for consistency. I opened https://github.com/WordPress/gutenberg/pull/28278 that will restrict such usage.